src/main/java/org/spongepowered/common/mixin/core/entity/item/EntityItemMixin.java

```diff
@@ -192,6 +192,22 @@
             ci.cancel();
         }
     }
+    
+    @Inject(
+        method = "onUpdate", 
+        at = @At (
+            value = "INVOKE", 
+            ordinal = 1, 
+            target = "Lnet/minecraft/entity/item/EntityItem;setDead()V")
+    )
+    private void fireExpireEntityEventTargetItem (final CallbackInfo ci) {
```

Comment:
I hate to be the one to complain about formatting...

This method name needs to be prefixed with `spongeImpl$` to be consistent with the class (though, @gabizou, [I thought the policy was to prefix `impl$`, not `spongeImpl$`](https://github.com/SpongePowered/SpongeCommon/wiki/Implementation-Policy:-Bridges-and-Mixins#rules-3))? 

Also, very nitpicky, but while you're there, can you remove the space between the method name and parameters, and the space after the method name?